include/llvm/Transforms/IPO/StructFieldCacheAnalysis.h

```diff
@@ -0,0 +1,30 @@
+//===-- StructFieldCacheAnalysis.h - Performs Cache Aware Structure Analysis- C++ -*-===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+//
+// This pass exports all llvm.bitset's found in the module in the form of a
```

Comment:
Oh, I am sorry that I forgot to change the comment in the header file. I copied it from another pass.